src/modules/ie-objects/helpers/convert-objects-to-xml.ts

```diff
@@ -5,9 +5,11 @@ import { IeObject } from '../ie-objects.types';
 export const convertObjectsToXml = (objects: Partial<IeObject>[]): string => {
```

Comment:
Q: was the spec file reviewed after these code changes to be sure they still succeed?
REQ: review test case and update were necessary